app/components-react/root/NotificationsArea.tsx

```diff
@@ -88,7 +88,7 @@ class NotificationsModule {
   clickNotif() {
     if (!this.currentNotif) return;
     Services.NotificationsService.actions.applyAction(this.currentNotif.id);
-    Services.NotificationsService.actions.markAsRead(this.currentNotif.id);
+    Services.NotificationsService.actions.showNotifications();
```

Comment:
I think this will open the notifications window on all notifications.  I think we only want to do this for notifications that don't have any actions associated with them.

So for example:
Notification with action - 1) apply action, 2) mark as read
Notification without action - 1) show notifications window 2) mark as read

The reason for this behavior is that some notifications actually have an action that opens up a child window themselves.  This is mutually exclusive with opening up the notifications window.

Reply:
Thanks for the clarification! I'll revert.